### DISCUSSION TITLE
Documentation does not link to latest version in dropdown

### DISCUSSION THREAD

**Discussion by kim3-sudo** [twbs/bootstrap#39430] [General]
Not sure where this goes, so putting here.

On https://getbootstrap.com/docs/5.2 (or any version that's not 5.3), there is a banner informing readers that there's a new version of Bootstrap available. However, the dropdown at the top-right still says that 5.2 is the latest version.

![Capture](https://github.com/twbs/bootstrap/assets/54730100/d7d2baba-cacb-4148-8d59-0ff68868d794)

**Comment by julien-deramond** [comment]
Yep, it's disturbing. It's identified in Bootstrap's backlog but this is not tackled as we haven't found the right solution yet.
